test/elm/external/external-test.ts

```diff
@@ -1,8 +1,9 @@
 import should from 'should';
 import 'should-sinon';
 import sinon from 'sinon';
-import { Code, Interval, Repository, RetrieveDetails, ValueSet } from '../../../src/cql';
+import { Code, Interval, Repository, RetrieveDetails } from '../../../src/cql';
 import setup from '../../setup';
+import { ValueSet } from '../../../src/datatypes/clinical';
```

Comment:
In this file, you import `ValueSet` from `src/datatypes/clinical`, but in `test/util/immutableUtil-test.ts`, you just change the import from `ValueSet` to `CQLValueSet` from `src/cql`. Is this difference intentional?